### PR TITLE
Add mac and linux script

### PR DESCRIPTION
**Pull request by ChrisForti**

### CODE REVIEW

File: myscript.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+x#!/bin/zsh
```
Comment (paulh6819) [inline myscript.sh]: please remove this "X"

File: myscript.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# shell script to install homebrew
+
+echo "checking if brew is installed"
+
+if (which brew)                   
+then 
+  echo "brew is installed"
+else
+  echo "installing brew"
+  /bin/bash -c "$(curl -fsSL
```
Comment (paulh6819) [inline myscript.sh]: line 12 and 13 have an unnecessary line break. please fix thank you

File: myscript.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  echo "brew is installed"
+else
+  echo "installing brew"
+  /bin/bash -c "$(curl -fsSL 
+https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+fi
+
+# checking if nano is installed
+# if nano installed
+# then echo "nano is already installed"
```
Comment (paulh6819) [inline myscript.sh]: confused about how this code ended. Please elaborate.

**Review by paulh6819 — CHANGES_REQUESTED** [review]

**Review by stephenlauck — APPROVED** [review]

**Review by JoshRodstein — APPROVED** [review]
LGTM

**Review by paulh6819 — APPROVED** [review]

**Pull request merged**